modules/ROOT/partials/configuration/autocorrect_capitalize.adoc

```diff
@@ -0,0 +1,24 @@
+[[autocorrect_capitalize]]
+== `+autocorrect_capitalize+`
+
+This option enables or disables capitalization autocorrection when the {productname} editor is loaded.
+
+When set to `true`, capitalization autocorrection is enabled when {productnam} is initialized.
```

Comment:
Should be `{productname}` not `{productnam}`

Reply:
Typo corrected.